unixfs.go

```diff
@@ -71,7 +71,7 @@ type UnixfsAPI interface {
 	//
 	// Note that some implementations of this API may apply the specified context
 	// to operations performed on the returned file
-	Get(context.Context, path.Path) (files.Node, error)
+	Get(context.Context, path.Path, bool) (files.Node, error)
```

Comment:
Could you add a description about what this bool flag means?

Reply:
Done